From 21788be688de90d4f12edb9f45967a481801bd5e Mon Sep 17 00:00:00 2001
From: Yuki Joou <yukijoou@kemonomimi.gay>
Date: Sun, 22 Jun 2025 11:48:55 +0200
Subject: [PATCH] Revert "Properly Align menubar when there is a sidebar
 drawer"

This reverts commit 640985886bc48fbbbe7b858cd2dd966d9edea431.

It seems another bit of the code is now handling the menubar, and having this
makes the menubar go off-screen when using a drawer, like in the case of merkuro
---
 src/controls/AbstractApplicationWindow.qml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/controls/AbstractApplicationWindow.qml b/src/controls/AbstractApplicationWindow.qml
index cbcec9c6..23bbbb34 100644
--- a/src/controls/AbstractApplicationWindow.qml
+++ b/src/controls/AbstractApplicationWindow.qml
@@ -243,11 +243,6 @@ QQC2.ApplicationWindow {
     contentItem.anchors.leftMargin: root.globalDrawer && root.globalDrawer.modal === false && (!root.pageStack || root.pageStack.leftSidebar !== root.globalDrawer) ? root.globalDrawer.width * root.globalDrawer.position : 0
     contentItem.anchors.rightMargin: root.contextDrawer && root.contextDrawer.modal === false ? root.contextDrawer.width * root.contextDrawer.position : 0
 
-    Binding {
-        target: root.menuBar
-        property: "x"
-        value: -contentItem.x
-    }
     Binding {
         target: root.header
         property: "x"
-- 
2.50.0

